recipes/spectrum/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.6.4" %}
+{% set name = "spectrum" %}
+{% set sha256 = "7d8e3fe074b2026f61b6539c4f847d0cb1dbed41d5c6788f6100bb5884aa65ec" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/43/a4/c85957430db53744fd70ce726d8cd0000ee33e18da39a8132ca4e5cdb2f5/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0 
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - matplotlib
+    - numpy
+    - scipy
+    - easydev
+  run:
+    - python
+    - matplotlib
+    - numpy
+    - scipy
+    - easydev
+
+test:
+  imports:
+    - spectrum
+
+about:
+  home: http://github.com/cokelaer/spectrum
+  license: LGPL v3+
+  license_family: LGPL
+  license_file: LICENSE
```

Comment:
It doesn't look like the license file is packaged with the source distribution. Please include it manually:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

and put in either an issue or a pull request upstream to include it via the `MANIFEST.in`. Please add the url for the issue as a comment in the recipe and xref the link here for future reference when upstream gets updated.